app/src/main/java/org/mozilla/tv/firefox/webrender/WebRenderFragment.kt

```diff
@@ -166,50 +100,9 @@ class WebRenderFragment : EngineViewLifecycleFragment(), Session.Observer {
         val layout = inflater.inflate(R.layout.fragment_browser, container, false)
 
         cursor = CursorController(this, cursorParent = layout.browserFragmentRoot,
-                view = layout.cursorView)
+                view = layout.cursorView, screenControllerStateMachine = serviceLocator!!.screenControllerStateMachine)
```

Comment:
nit: layout.serviceLocator (or whatever is in between): you should use the view context instead of !!.